oamapps/postConfigure/postConfigure.cpp

```diff
@@ -75,6584 +75,91 @@
-            pmnextbrick[pm]++;
-        }
-
-        command += "force >> /tmp/glusterCommands.txt 2>&1";
-        cout << "Gluster create and start volume dbroot" << oam.itoa(dbrootID) << "...";
-        status = system(command.c_str());
-
-        if (WEXITSTATUS(status) != 0 )
-        {
-            if (oam.checkLogStatus("/tmp/glusterCommands.txt", "dbroot" + oam.itoa(dbrootID) + " already exists" ))
-            {
-                string errmsg1;
-                string errmsg2;
-                int ret = oam.glusterctl(oam::GLUSTER_DELETE, oam.itoa(dbrootID), errmsg1, errmsg2);
-
-                if ( ret != 0 )
-                {
-                    cerr << "FAILURE: Error replacing existing gluster dbroot# " + oam.itoa(dbrootID) + ", error: " + errmsg1 << endl;
-                    exit(1);
-                }
-
-                status = system(command.c_str());
-
-                if (WEXITSTATUS(status) != 0 )
-                {
-                    cout << "ERROR: Failed to create volume dbroot" << oam.itoa(dbrootID) << endl;
-                    exit(1);
-                }
-            }
-            else
-            {
-                cout << "ERROR: Failed to create volume dbroot" << oam.itoa(dbrootID) << endl;
-                exit(1);
-            }
-        }
-
-        if (rootUser)
-        {
-            command = "gluster volume start dbroot" + oam.itoa(dbrootID) + " >> /tmp/glusterCommands.txt 2>&1";
-            status = system(command.c_str());
-
-            if (WEXITSTATUS(status) != 0 )
-            {
-                cout << "ERROR: Failed to start dbroot" << oam.itoa(dbrootID) << endl;
-                exit(1);
-            }
-        }
-        else
-        {
-            int user = getuid();
-            int group = getgid();
-            command = "sudo gluster volume set dbroot" + oam.itoa(dbrootID) + " storage.owner-uid " + oam.itoa(user) + " >> /tmp/glusterCommands.txt 2>&1";;
-            status = system(command.c_str());
-
-if (WEXITSTATUS(status) != 0)
-{
-    cout << "ERROR: Failed to start dbroot" << oam.itoa(dbrootID) << endl;
-    exit(1);
-}
-
-command = "sudo gluster volume set dbroot" + oam.itoa(dbrootID) + " storage.owner-gid " + oam.itoa(group) + " >> /tmp/glusterCommands.txt 2>&1";;
-status = system(command.c_str());
-
-if (WEXITSTATUS(status) != 0)
-{
-    cout << "ERROR: Failed to start dbroot" << oam.itoa(dbrootID) << endl;
-    exit(1);
-}
-
-command = "sudo gluster volume start dbroot" + oam.itoa(dbrootID) + " >> /tmp/glusterCommands.txt 2>&1";
-status = system(command.c_str());
-
-if (WEXITSTATUS(status) != 0)
-{
-    cout << "ERROR: Failed to start dbroot" << oam.itoa(dbrootID) << endl;
-    exit(1);
-}
-        }
-
-        cout << "DONE" << endl;
-    }
-
-    cout << endl << "----- Data Redundancy Configuration Complete -----" << endl << endl;
+bool updateBash(string homeDir)
```

Comment:
this function already existed pre-cleanup of postConfig. just updated slighted to take in an argument rather than use a global variable